.travis.yml

```
@@ -1,11 +1,14 @@
 language: python
+python:
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  - 3.7
```

Python 3.7 is not stable on travis, see https://github.com/travis-ci/travis-ci/issues/9815
```suggestion
  - 3.7
    dist: xenial
    sudo: true
```